acdc-ws/app/utils/MetricReporter.scala

```diff
@@ -0,0 +1,105 @@
+package utils
+
+import io.prometheus.client._
+import io.prometheus.client.exporter.common.TextFormat
+import io.prometheus.client.hotspot.DefaultExports
+
+import java.io.StringWriter
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.jdk.CollectionConverters._
+
+object Metrics {
+
+  val DefaultBucketValues: Array[Double] = Array(
+    .005,
+    .010,
+    .025,
+    .050,
+    .100,
+    .250,
+    .500,
+    1.0,
+    2.5,
+    5.0,
+    10.0,
+    15.0,
+    60.0,
+    300.0,
+    1200.0,
+    Double.PositiveInfinity
+  )
+
+  val httpDurationSeconds: Histogram = Histogram.build
+    .buckets(DefaultBucketValues: _*)
+    .name("http_request_duration_seconds")
+    .help("Total HTTP Requests Count")
```

Comment:
this help was copied from httpTotalRequest.  Edit it to something like "Http Request Duration in seconds"

Reply:
Good catch.